yarn.lock

```diff
@@ -5497,9 +5497,10 @@ bitcoinjs-lib@^6.0.0:
     varuint-bitcoin "^1.1.2"
     wif "^2.0.1"
 
-"bitcoinjs-lib@git+https://github.com/BitGo/bitcoinjs-lib#1afb49753714b92f1078bf2a7afe6a5ef3280e2d":
-  version "7.0.0-rc.0"
-  resolved "git+https://github.com/BitGo/bitcoinjs-lib#1afb49753714b92f1078bf2a7afe6a5ef3280e2d"
+"bitcoinjs-lib@npm:@bitgo/bitcoinjs-lib@7.0.0-rc.1":
+  version "7.0.0-rc.1"
+  resolved "https://artifactory.bitgo-ci.com/artifactory/api/npm/npm/@bitgo/bitcoinjs-lib/-/bitcoinjs-lib-7.0.0-rc.1.tgz#a2ce0b0206dcb20fc0534c346fd14b27e620f782"
```

Comment:
Need to point to npm.

Reply:
asdfk thanks, I swear I put a .npmrc to prevent just this today.

Reply:
Good catch @mmcshinsky-bitgo